modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -130,6 +140,14 @@ public AspNetCoreServerCodegen() {
                 CodegenConstants.SOURCE_FOLDER_DESC,
                 sourceFolder);
 
+        addOption(CodegenConstants.API_PACKAGE,
+                CodegenConstants.API_PACKAGE_DESC,
+                apiPackage);
+
+        addOption(CodegenConstants.MODEL_PACKAGE,
+                CodegenConstants.MODEL_PACKAGE_DESC,
+                modelPackage);
+
```

Comment:
@A-Joshi If I remember it correctly, these are global options so we shouldn't need to add it to the generator manually?

Reply:
Did not realize that - removed the additional code.